### PR TITLE
Made changes to documentation and added linting tests

### PR DESCRIPTION
**Pull request by jselmani**
I made corrections to your README where you said:
`git start`

I changed that to:
`npm start`

I also added linting tests based on your projects style.  Feel free to change those as you wish but since you included the module for `eslint` I thought it would be good practice to include them for you. 😃

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 var app = require('./app');
 
 app.listen(8000, function() {
-  console.log('Server started on http://localhost:8000');
+  //console.log('Server started on http://localhost:8000');
```
Comment (ericschv) [inline src/server.js]: Need the console.log to let users know what address to go to in order to view the application in the browser.

File: README.md [hunk 2]
```diff
@@ -1,41 +1,41 @@
 # node-extract-file-info  <img src="https://travis-ci.org/ericschv/node-extract-file-info.svg?branch=master">
-This library contains functions for node applications that focus on extracting information about a file being servered to the browser. The goal is to create a library that is RESTful and can be used in a node web service.
-  
+This library contains functions for Node applications that focus on extracting information about a file being served to the browser. The goal is to create a library that is RESTful and can be used in a node web service.
+
 ## How to setup
-In order to get up and running you to have node installed. Once you have that setup, clone this repository  
+In order to get up and running you to have Node installed. This project was built using Node version 8.9.1.  Once you have that setup, clone the following repository:
```
Comment (ericschv) [inline README.md]: project was built using Node 6.11.2

**Comment by ericschv** [comment]
@jselmani good job!

**Comment by jselmani** [comment]
Thanks @ericschv 👍

**Pull request merged**